lib/puppet/provider/service/systemd.rb

```diff
@@ -7,6 +7,7 @@
 
   defaultfor :osfamily => [:archlinux]
   defaultfor :osfamily => :redhat, :operatingsystemmajrelease => "7"
+  defaultfor :osfamily => :redhat, :operatingsystem => :fedora, :operatingsystemmajrelease => ["17", "18", "19", "20", "21"]
```

Comment:
@peterhuene that would be a nicer way to deal with this. Given that we don't have this capability yet, is that your only feedback? If so, I'd like to get this merged.